econet_electric_tank_water_heater.yaml

```diff
@@ -39,6 +39,8 @@ sensor:
     sensor_datapoint: LOHTRTMP
     unit_of_measurement: "°F"
     accuracy_decimals: 1
+    filters:
+      - throttle_average: ${econet_throttle_interval}
```

Comment:
See https://github.com/esphome-econet/esphome-econet/issues/472#issuecomment-2781373454
I don't see how this would throttle.